templates/boltzmann.js

```diff
@@ -1149,7 +1156,12 @@ function applyHeaders (headers = {}) {
   }
 }
 
-const applyXFO = (mode) => applyHeaders({ 'x-frame-options': mode })
+const applyXFO = (mode) => {
+  if (!['DENY', 'SAMEORIGIN'].includes(mode)) {
+    throw new Error('applyXFO(): Allowed x-frame-options directives are DENY and SAMEORIGIN.')
```

Comment:
How would you feel about linking to the boltzmann reference docs at the current boltzmann version in these errors?